htdocs/compta/resultat/result.php

```diff
@@ -353,7 +353,7 @@
 						$r = 0;
 					}
 
-					print '<td class="liste_total right"><span class="amount">'.price($r).'</span></td>';
+					print '<td class="liste_total right"><span class="amount">'.price($r, 0, '', 1, 0, 0).'</span></td>';
```

Comment:
On my side, what i need is real value. So if amount is 1.12 i want to see 1.12 so i can compare value to other report to validate my accountancy. 
Truncing data would surely generate bug report since value ill be wrong.
So yes, please, add a hidden option for this. I suggest this name:
ACCOUNTANCY_TRUNC_DECIMAL_ON_BALANCE_REPORT

Reply:
Done